### PR TITLE
fluent-package v5.0.3 has been released

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
TODO: add note about delaying windows.

**Comment by kenhys** [comment]
Added note for Windows.

**Comment by daipom** [comment]
Thanks! I'm seeing.

File: content/blog/20240329_fluent-package-v5.0.3-has-been-released.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# fluent-package v5.0.3 has been released
+
+Hi users!
+
+We have released fluent-package [v5.0.3](https://github.com/fluent/fluent-package-builder/releases/tag/v5.0.3)
+fluent-package is a stable distribution package of Fluentd. (successor of td-agent v4)
```
Comment (daipom) [inline content/blog/20240329_fluent-package-v5.0.3-has-been-released.md:6]: This becomes one line, so we should add `.` or insert an empty line.

File: content/blog/20240329_fluent-package-v5.0.3-has-been-released.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+This is a maintenance release of v5.0.x LTS series.
+As significant slow starting service and crash issues during startup on Windows were fixed, we recommend upgrading to fluent-package v5.0.3!
+
+### Changes from fluent-package v5.0.2
+
+* Update fluentd to [1.16.5](https://github.com/fluent/fluentd/releases/tag/v1.16.5)
+  See [Fluentd v1.16.5 has been released](/blog/fluentd-v1.16.5-has-been-released) blog article about details.
```
Comment (daipom) [inline content/blog/20240329_fluent-package-v5.0.3-has-been-released.md:14]: Ditto.

File: content/blog/20240329_fluent-package-v5.0.3-has-been-released.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* Update bundled plugins
+  * e.g. fluent-diagtool v1.0.5. It supports to collect list of plugins on Windows.
+* msi: fixed wrong environment path for Fluent Package Prompt ([#606](https://github.com/fluent/fluent-package-builder/pull/606))
+  * It breaks fluent-diagtool behavior to launch fluent-gem correctly.
+* msi: removed unnecessary path delimiter ([#607](https://github.com/fluent/fluent-package-builder/pull/607))
+  * It doesn't cause any problem yet, but it should treat `%~dp0~ correctly.
```
Comment (daipom) [inline content/blog/20240329_fluent-package-v5.0.3-has-been-released.md:20]:
```suggestion
  * It doesn't cause any problem yet, but it should treat `%~dp0` correctly.
```

File: content/blog/20240329_fluent-package-v5.0.3-has-been-released.md (hunk 1 above)
Comment (kenhys) [reply content/blog/20240329_fluent-package-v5.0.3-has-been-released.md]: Thanks!
I've fixed it.

File: content/blog/20240329_fluent-package-v5.0.3-has-been-released.md (hunk 2 above)
Comment (kenhys) [reply content/blog/20240329_fluent-package-v5.0.3-has-been-released.md]: ditto.

**Review by daipom — APPROVED** [review]
LGTM. Thanks!

`Note` is very cool!

**Pull request merged**